source/js/main.js

```diff
@@ -231,6 +232,42 @@ let main = {
     if (document.querySelector(`#news`)) {
       ReactDOM.render(<News env={env}/>, document.querySelector(`#news`));
     }
+
+    // Science Fellowship
+    if (document.getElementById(`featured-science-fellow`)) {
+      let metadata = {
+        featured: true,
+        'internet_health_issues': [ `Decentralization`, `Open Innovation` ],
+        links: [],
+        name: `Firstname Surname`,
+        role: `[Area Fellowship] Fellow, [Year]`,
+        location: `City, Country`,
+        image: `https://images.pexels.com/photos/264206/pexels-photo-264206.jpeg?w=500`,
+        quote: `Quote quote quote quote quote quote quote quote quote quote quote quote.`,
+        affiliations: [ `Stanford University Professor; YouthLAB founder`],
+        'fellow_directory_link': { type: `science`, link: `/fellowships/directory` }
+      };
+
+      ReactDOM.render(<Person metadata={metadata} />, document.getElementById(`featured-science-fellow`));
+    }
+
+    // Open Web Fellowship
+    if (document.getElementById(`featured-open-web-fellow`)) {
+      let metadata = {
+        featured: true,
+        'internet_health_issues': [ `Decentralization`, `Open Innovation` ],
+        links: [],
+        name: `Firstname Surname`,
+        role: `[Area Fellowship] Fellow, [Year]`,
+        location: `City, Country`,
+        image: `https://images.pexels.com/photos/264206/pexels-photo-264206.jpeg?w=500`,
+        quote: `Quote quote quote quote quote quote quote quote quote quote quote quote.`,
+        affiliations: [ `Stanford University Professor; YouthLAB founder`],
+        'fellow_directory_link': { type: `science`, link: `/fellowships/directory` }
+      };
+
+      ReactDOM.render(<Person metadata={metadata} />, document.getElementById(`featured-open-web-fellow`));
+    }
```

Comment:
This feels WETter than necessary...could the metadata be passed in as props on the target div (in the markup), and then render run on a generic class?